### PR TITLE
feat: monaco command supports and add Masa.Try

### PR DESCRIPTION
**Pull request by 239573049**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@capdiem PTAL

File: src/Masa.Try.Shared/Pages/Index.razor.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+using Microsoft.CodeAnalysis;
+using Microsoft.JSInterop;
+
+namespace Masa.Try.Shared.Pages;
+
+public partial class Index
+{
+    private Type? componentType;
+
+    private MMonacoEditor _Monaco;
```
Comment (capdiem) [inline src/Masa.Try.Shared/Pages/Index.razor.cs:13]:
```suggestion
    private MMonacoEditor _monaco;
```

File: src/Masa.Try.Shared/Pages/Index.razor.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    {
+        RazorCompile.Initialized(await GetReference(), GetRazorExtension());
+
+        await base.OnInitializedAsync();
+    }
+
+    private async Task InitMonaco()
+    {
+        // 监听CTRL+S
+        await _Monaco.AddCommand(2097, DotNetObjectReference.Create(this), nameof(RunCode));
```
Comment (capdiem) [inline src/Masa.Try.Shared/Pages/Index.razor.cs:92]: `DotNetObjectReference.Create(this)` needs dispose

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by capdiem — CHANGES_REQUESTED** [review]

File: src/Masa.Try.Shared/Pages/Index.razor.cs [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        {
+            razorExtension.Add(new AssemblyExtension(asm.FullName, AppDomain.CurrentDomain.Load(asm.FullName)));
+        }
+
+        return razorExtension;
+    }
+
+    public void Dispose()
+    {
+        DotNetObject.Dispose();
```
Comment (capdiem) [inline src/Masa.Try.Shared/Pages/Index.razor.cs:138]:
```suggestion
        _objRef.Dispose();
```

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by capdiem — APPROVED** [review]

**Pull request merged**